### ISSUE TITLE
Vertical/Side Nav - Child page links should probably have indentation 

### ISSUE THREAD

**Issue by mmmavis** [mozilla/foundation.mozilla.org#2418]
<img width="375" alt="image" src="https://user-images.githubusercontent.com/2896608/50798003-98833300-128b-11e9-950d-25b30269c200.png">

- https://foundation.mozilla.org/en/opportunity/web-literacy/
- https://foundation.mozilla.org/en/opportunity/mozilla-open-leaders/round-7/schedule/

Not sure if adding indentation would be the solution but having arrows on child page nav links doesn't seem very conventional to me.

**Comment by mmmavis** [comment]
@kristinashu what should we do to close this ticket? Is simply adding indentation enough?

**Comment by kristinashu** [comment]
I think it might be a little more complicated then that. We also might want to have less space between tertiary nav items, and need to decided if we need the arrows or something else or not. 

To close this ticket we need a comp of this and for it to go through design review. Once that is done we can open an implementation ticket.

**Comment by cadecairos** [comment]
@kristinashu Should we keep this open or file a new ticket based on the ongoing IA work?

**Comment by kristinashu** [comment]
I think we can keep this as is. The new IA shouldn't affect the need for this side nav.

**Comment by kristinashu** [comment]
Closing this since this page type gets used less and less now that we have the Publications/Article page type.
